proto/connectrpc/conformance/v1/service.proto

```diff
@@ -110,6 +110,8 @@ message UnaryResponseDefinition {
     // Response data to send
     bytes response_data = 2;
     // Error to raise instead of response message
+    // If no error details are explicitly provided to return, servers should
+    // still build a ConformancePayload and set it as the details.
```

Comment:
This wording is unclear. It implies to me that the error should always include a `ConformancePayload` (probably the use of the word "still"). But the implementation only sets this payload if the details are empty.

FWIW, I think the server should just always add the request info. If the details are not empty, it can append the request info.